### PR TITLE
Add metrics for plotting

### PR DESCRIPTION
**Pull request by kissken**
we need some metric to track how long we draws plots

### CODE REVIEW

**Review by Tetrergeru — APPROVED** [review]

File: metrics/notifier.go [hunk 1]
```diff
@@ -2,26 +2,30 @@ package metrics
...
-		SendersFailedMetrics:   NewMetersCollection(registry),
+		SubsMalformed:                  registry.NewMeter("subs", "malformed"),
+		EventsReceived:                 registry.NewMeter("events", "received"),
+		EventsMalformed:                registry.NewMeter("events", "malformed"),
+		EventsProcessingFailed:         registry.NewMeter("events", "failed"),
+		EventsByState:                  NewMetersCollection(registry),
+		SendingFailed:                  registry.NewMeter("sending", "failed"),
+		SendersOkMetrics:               NewMetersCollection(registry),
+		SendersFailedMetrics:           NewMetersCollection(registry),
+		PlotsBuildDurationMs:           registry.NewMeter("plots", "build_duration_ms"),
```
Comment (Tetrergeru) [inline metrics/notifier.go:28]: Maybe we should write words as separate strings here?

```suggestion
		PlotsBuildDurationMs:           registry.NewMeter("plots", "build", "duration", "ms"),
```

File: metrics/prometheus.go [hunk 2]
```diff
@@ -47,7 +47,12 @@ func (source *PrometheusRegistryAdapter) NewCounter(path ...string) Counter {
 }
 
 func (source *PrometheusRegistryAdapter) NewHistogram(path ...string) Histogram {
-	var histogramOpts = prometheus.HistogramOpts{Namespace: namespace, Subsystem: source.service, Name: getPrometheusMetricName(path)}
+	var histogramOpts = prometheus.HistogramOpts{
+		Namespace: namespace,
+		Subsystem: source.service,
+		Name:      getPrometheusMetricName(path),
+		Buckets:   []float64{0.005, 0.01, 0.025, 0.05, 0.075, 0.1, 0.25, 0.5, 0.75, 1, 2.5, 5, 7.5, 10, 20, 100, 200, 300, 500, 1000},
```
Comment (kissken) [inline metrics/prometheus.go:54]: @Tetrergeru тут в целом на удачу бакетов отсыпала

File: metrics/notifier.go (hunk 1 above)
Comment (kissken) [reply metrics/notifier.go]: fixed

File: metrics/prometheus.go (hunk 2 above)
Comment (Tetrergeru) [reply metrics/prometheus.go:54]: 👌

**Review by Tetrergeru — DISMISSED** [review]

**Review by Tetrergeru — APPROVED** [review]

**Review by Tetrergeru — APPROVED** [review]

**Review by Tetrergeru — APPROVED** [review]

**Pull request merged**